source/content/visual-studio-code.md

```diff
@@ -44,25 +46,25 @@ wp-content/upgrade/
 
 ```
 
-At this point, you should see your site files in the File Explorer on the left side of VS Code, and you can open any of these for editing in the main window by clicking on the file name.
+At this point, you should see your site files in the File Explorer on the left side of VS Code. Click the file name to open it for editing in the main window.
 
-## Deploy your local code changes to a Pantheon environment
+## Deploy local code changes to a Pantheon environment
 
 Once you've made some code changes, you'll want to push those up to your Pantheon Dev or Multidev environment. You can do this using Git, or with SFTP.
 
-### Push your changes with Git (recommended)
+### Push changes with Git (recommended)
 VS Code includes Git integration and an integrated terminal, either of which can be used to make Git changes and push them back to your Pantheon repository and Dev environment.
 
-Make sure your site is in [Git mode](//guides/quickstart/connection-modes) before pushing code via Git.
+Make sure your site is in [Git mode](/guides/quickstart/connection-modes) before pushing code via Git.
 
 #### [Git Version Control in VS Code](https://code.visualstudio.com/docs/introvideos/versioncontrol)
-Open the Git menu (**Ctrl/Command** + **Shift** + **G** or **View** -> **SCM**). From here you can review your outstanding changes, commit them with a message, switch branches and more. See the video for more information.
+Open the Git menu (**Ctrl** + **Shift** + **G** or **View** -> **SCM**). From here you can review your outstanding changes, commit them with a message, switch branches and more. See the [video on the VS Code docs](https://code.visualstudio.com/docs/introvideos/versioncontrol) for more information.
 
 #### [Integrated Terminal](https://code.visualstudio.com/docs/editor/integrated-terminal)
 Open the integrated terminal with **Ctrl** + **`** or **View** -> **Terminal**. Now you can interact with Git as you would normally, without leaving VS Code.
 
 ### Upload your changes with SFTP
-The SFTP extension for VS Code allows developers to upload code to your Pantheon Dev or Multidev environment directly from VS Code, as well as downloading files from the Pantheon servers.
+The SFTP extension for VS Code allows developers to upload code to the Pantheon Dev or Multidev environment directly from VS Code, as well as download files from the Pantheon servers.
```

Comment:
IDG the change from `your` to `the` here.

Reply:
I figured it reads better without the 2nd-person voicing. If 2nd-person is better, though, maybe it should be, "The SFTP extension for VS Code allows _you_ to upload code to your Pantheon Dev..."

I don't have a real preference here, whatever best matches the tone of the rest of the Docs is cool by me.

Reply:
WFM!